static/app/components/performance/searchBar.tsx

```diff
@@ -193,6 +193,12 @@ function SearchBar(props: SearchBarProps) {
   const handleSearch = (query: string, asRawText: boolean) => {
     setSearchResults([]);
     setSearchString(query);
+
+    // Transactions with a space must be wrapped in quotes
+    if (query.includes(' ')) {
+      query = `"${query}"`;
+    }
```

Comment:
There are other cases where this won't handle it correctly. For example, if the transaction name contains quotes. You may want to look into the `MutableSearch` class.